tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -251,8 +251,14 @@ object NativeConfig {
     def withCompileOptions(value: Seq[String]): NativeConfig =
       copy(compileOptions = value)
 
-    def withTargetTriple(value: Option[String]): NativeConfig =
+    def withTargetTriple(value: Option[String]): NativeConfig = {
+      val propertyName = "target.triple"
```

Comment:
If this data is already in the `targetTriple`, why are we setting a system property? I am a little late reviewing this so I may have missed something.

Reply:
Sorry for the delayed reply.
This is not quite right, as you suspected.   I hope to submit another PR soon.